tests/foreman/cli/test_oscap.py

```diff
@@ -653,75 +634,75 @@ def test_positive_associate_scap_policy_with_tailoringfiles(self):
-                        'period': OSCAP_PERIOD['weekly'].lower(),
-                        'weekday': OSCAP_WEEKDAY['friday'].lower(),
-                        'tailoring-file': tailoring_file_a['name'],
-                        'tailoring-file-profile-id': tailoring_file_profile_a_id,
-                    }
-                )
-                assert scap_policy['deployment-option'] == deploy
-                assert scap_policy['tailoring-file-id'] == tailoring_file_a['id']
-                assert scap_policy['tailoring-file-profile-id'] == tailoring_file_profile_a_id
-
-                Scappolicy.update(
-                    {
-                        'name': scap_policy['name'],
-                        'tailoring-file': tailoring_file_b['name'],
-                        'tailoring-file-profile-id': tailoring_file_profile_b_id,
-                    }
-                )
-                scap_info = Scappolicy.info({'name': scap_policy['name']})
-                assert scap_info['tailoring-file-id'] == tailoring_file_b['id']
-                assert scap_info['tailoring-file-profile-id'] == tailoring_file_profile_b_id
-
-                Scappolicy.delete({'name': scap_policy['name']})
-                with pytest.raises(CLIReturnCodeError):
-                    Scapcontent.info({'name': scap_policy['name']})
-
-                scap_policy = make_scap_policy(
-                    {
-                        'scap-content-id': self.scap_id_rhel7,
-                        'deploy-by': deploy,
-                        'scap-content-profile-id': self.scap_profile_id_rhel7,
-                        'period': OSCAP_PERIOD['weekly'].lower(),
-                        'weekday': OSCAP_WEEKDAY['friday'].lower(),
-                        'tailoring-file-id': tailoring_file_a['id'],
-                        'tailoring-file-profile-id': tailoring_file_profile_a_id,
-                    }
-                )
-                assert scap_policy['deployment-option'] == deploy
-                assert scap_policy['tailoring-file-id'] == tailoring_file_a['id']
-                assert scap_policy['tailoring-file-profile-id'] == tailoring_file_profile_a_id
-
-                Scappolicy.update(
-                    {
-                        'id': scap_policy['id'],
-                        'tailoring-file-id': tailoring_file_b['id'],
-                        'tailoring-file-profile-id': tailoring_file_profile_b_id,
-                    }
-                )
-                scap_info = Scappolicy.info({'id': scap_policy['id']})
-                assert scap_info['tailoring-file-id'] == tailoring_file_b['id']
-                assert scap_info['tailoring-file-profile-id'] == tailoring_file_profile_b_id
-
-                Scappolicy.delete({'id': scap_policy['id']})
-                with pytest.raises(CLIReturnCodeError):
-                    Scapcontent.info({'name': scap_policy['name']})
 
+        scap_policy = make_scap_policy(
```

Comment:
I wonder if `make scap_policy` can be transformed to fixture, looks like it has been recused with same params multiple times.

Reply:
it's true that make_scap_policy has been reused sometimes.
But it's not the case for all tests, and I don't think we should convert it to fixture